src/NHibernate.Validator/Event/ValidatePreInsertEventListener.cs

```diff
@@ -58,9 +60,15 @@ public void Initialize(Configuration cfg)
 
 		#region IPreInsertEventListener Members
 
+		public Task<bool> OnPreInsertAsync(PreInsertEvent @event, CancellationToken cancellationToken)
+		{
+			OnPreInsertAsync(@event, cancellationToken);

```

Comment:
Stack overflow here.
Should be `OnPreInsert(@event);`.

Reply:
Shame to me
fixed